crates/biome_lsp/src/server.rs

```diff
@@ -132,7 +132,21 @@ impl LSPServer {
         capabilities.add_capability(
             "biome_did_change_workspace_settings",
             "workspace/didChangeWatchedFiles",
-            if let Some(base_path) = self.session.base_path() {
+            if let Some(folders) = self.session.get_workspace_folders() {
```

Comment:
I noticed that we weren't considering workspace folders when the configuration changed. This should fix some bugs.